client/src/component/PullRequestCard.js

```diff
@@ -0,0 +1,33 @@
+import { AppContext } from "../AppContext";
+import { useContext } from "react";
+import RoundProgressBar from "./RoundProgressBar";
+
+const PullRequestCard = () => {
+  const createdPullRequests = 20;
+  const requiredPullRequests = 20;
+  const currentMilestone = "Data Base week 3";
+  const nextMilestone = "Final project";
+  const remainingDays = 17;
+  const nextMilestonePullRequests = 24;
+
```

Comment:
good job , for creating this value , we can just move them later to another component and use them same as props